### ISSUE TITLE
Not able to loign by any account

### ISSUE THREAD

**Issue by hrshlmhjn** [cguZZman/plugin.googledrive#342]
This app is blocked This app tried to access sensitive info in your Google Account.
When trying to login on https://drive-login.herokuapp.com/ , after selecting account this above pop up comes up with no option to skip it.
![image](https://github.com/cguZZman/plugin.googledrive/assets/167405103/43e6c1d3-d132-4d5f-b3f8-fa9df854f889)

**Comment by Varun23102002** [comment]
Same issue I am facing

**Comment by Mfstick** [comment]
Facing same issue. Unable to access Gdrive.

**Comment by Favo1** [comment]
Also facing the same issue.

**Comment by Varun23102002** [comment]
Please guys comment the alternative way to play Google drive video files in phone and Tv

**Comment by alarno** [comment]
same issue here!

**Comment by cguZZman** [comment]
Hi all,
I checked my email and Google sent me one saying the application violates the "[AUP-3: Always ask permission](https://developers.google.com/photos/library/guides/acceptable-use#permission)" policy. :man_shrugging:

I will try to resolve it. It could take some time.

**Comment by alarno** [comment]
> Hi all, I checked my email and Google sent me one saying the application violates the "[AUP-3: Always ask permission](https://developers.google.com/photos/library/guides/acceptable-use#permission)" policy. 🤷‍♂️
> 
> I will try to resolve it. It could take some time.

This is great Carlos! If there is anything we can do to help, do not hesitate to ask!

**Comment by Nickytun** [comment]
I have same problem. Hope will be fixed it soon. Thanks for your hard work.

**Comment by alarno** [comment]
I've look around, and it seams google is limiting access with only username and password, [Control access to less secure apps](https://support.google.com/a/answer/6260879), if we can help @cguZZman in any way, I'm available

**Comment by Varun23102002** [comment]
I think we should use oauth client I'd and secret of our own to access Google drive it's a best and secure way @cguZZman

**Comment by alarno** [comment]
> I think we should use oauth client I'd and secret of our own to access Google drive it's a best and secure way @cguZZman

But maybe this is not possible or difficult to setup? Or maybe we have to download the code in our own gdrive account and then it would work??

**Comment by Varun23102002** [comment]
Ya bro it's tough but we can have our data safe and no one can access. If app blocked we can create as many we required

**Comment by alarno** [comment]
> Ya bro it's tough but we can have our data safe and no one can access. If app blocked we can create as many we required

Let me get this right, you are saying we should do this oauth system for @cguZZman code? How should we proceed?

**Comment by Varun23102002** [comment]
Check out this repo app https://github.com/itszechs/DriveStream it uses oauth system I am using from past 1½ Year

**Comment by alarno** [comment]
> Check out this repo app https://github.com/itszechs/DriveStream it uses oauth system I am using from past 1½ Year

but how can we use this as a kodi addon?

**Comment by Liva1525** [comment]
> oauth

Does this repo works in kodi

**Comment by Liva1525** [comment]
> > Check out this repo app https://github.com/itszechs/DriveStream it uses oauth system I am using from past 1½ Year
> 
How to use drive stream app

**Comment by Liva1525** [comment]
> Hi all,
> I checked my email and Google sent me one saying the application violates the "[AUP-3: Always ask permission](https://developers.google.com/photos/library/guides/acceptable-use#permission)" policy. :man_shrugging:
> 
> I will try to resolve it. It could take some time.

Any updates admin?

**Comment by kamillopl** [comment]
@Liva1525 why are you spamming?

What should I copy to a fresh Kodi installation from my Kodi with authenticated and working Google Drive addon to make it work there? It is possible or will it make some conflicts when using addon on 2 devices at the same time?

UPDATE:

I copied a _accounts.db_ file from _Kodi/userdata/addon_data/plugin.googledrive_ and it works. So if you have verified account somewhere, e.g. on your PC, and want to use this account on another device, then you can put there this file.
I haven't checked what will happend if I use that account on 2 devices at the same time. If this account will be verified separately it propably would work, but when I have copies, it may cause problems.

**Comment by Bhejan** [comment]
Fire TV stick 4k max Drive stream and kodi add account not working best alternative Google drive access files  slove this problem

**Comment by Bhejan** [comment]
Kodi Google drive addon add account show qr code and scan qr code redirect Google sign page and sing this Google account and show This error 
This app is blocked
This app tried to access sensitive info in your Google Account. To keep your account safe, Google blocked this access. Slove this problem step by step

**Comment by Liva1525** [comment]
Its almost one week admin unable to resolve the login issue is there any alternative addon please let me know

**Comment by Varun23102002** [comment]
Anyone have alternative method to stream Google drive videos in Android Tv

**Comment by iamat8080** [comment]
mixplore app can fix this using gdrive to cast on tv @Varun23102002 @Liva1525

**Comment by alarno** [comment]
> mixplore app can fix this using gdrive to cast on tv @Varun23102002 @Liva1525

can you explain how? I look it, and it is like a file explorer, not seems to stream video...

**Comment by Liva1525** [comment]
@cguZZman admin please reply us any updates?we are waiting for your reply when its going to be fixed

**Comment by Bhejan** [comment]
Google drive on TV how to step by step

**Comment by Bhejan** [comment]
Kodi not working Google drive addon please solve this problem as soon as possible

**Comment by Bhejan** [comment]
Kodi Google drive addon add account  show This error  slove this problem  step by step solution 
![IMG_20240430_222855](https://github.com/cguZZman/plugin.googledrive/assets/168219217/ac05ed61-5f04-45b9-ad92-9dc2f2f632d0)

**Comment by Bhejan** [comment]
Please suggest kodi alternative Google drive use TV

**Comment by Varun23102002** [comment]
@Bhejan try nplayer

**Comment by Liva1525** [comment]
> @Bhejan try nplayer

Did you try? Is it working fine?

**Comment by Liva1525** [comment]
Nplayer working smoother but EAC3 codec not supported and no audio sound

**Comment by Varun23102002** [comment]
@Liva1525 I myself patched nplayer with eac3 codec

**Comment by Liva1525** [comment]
> @Liva1525 I myself patched nplayer with eac3 codec

How you patched please help

**Comment by Varun23102002** [comment]
Give m telegram I'd I will send u apk file

**Comment by Varun23102002** [comment]
@Liva1525 give me telegram I'd I will send you apk file

**Comment by Liva1525** [comment]
> @Liva1525 give me telegram I'd I will send you apk file 

@nolaf1525

**Comment by Andrew-J-Larson** [comment]
> Anyone have alternative method to stream Google drive videos in Android Tv 

Use Solid Explorer on your phone, connect to Google Drive there, then cast from there

**Comment by Liva1525** [comment]
Not only kodi gdrive addon is blocked many other famous players are also blocked by Google  like mx player and gom player.. maybe nplayer also going to get blocked

**Comment by alarno** [comment]
> Not only kodi gdrive addon is blocked many other famous players are also blocked by Google like mx player and gom player.. maybe nplayer also going to get blocked

you are right, nplayer also bloqued google account on ipad, we have to wait any skilled developer fixed, maybe with oauth...

**Comment by Andrew-J-Larson** [comment]
Guys, stop bugging the dev. They already know the problem, and are working on a fix.

If you need a work around, I suggest using a file manager on your phone, such as [Solid Explorer](https://play.google.com/store/apps/details?id=pl.solidexplorer2), that has Google Drive connection features and video casting built-in. You can sign into your Google account in the app, and then cast the videos from there to your affected TV device, until this is sorted out.

(normally I would suggest using Solid Explorer directly on the TV, but the latest version of Google TV has broken Google Drive logins for the TV version of the app)

**Comment by Liva1525** [comment]
> Guys, stop bugging the dev. They already know the problem, and are working on a fix.
> 
> If you need a work around, I suggest using a file manager on your phone, such as [Solid Explorer](https://play.google.com/store/apps/details?id=pl.solidexplorer2), that has Google Drive connection features and video casting built-in. You can sign into your Google account in the app, and then cast the videos from there to your affected TV device, until this is sorted out.
> 
> (normally I would suggest using Solid Explorer directly on the TV, but the latest version of Google TV has broken Google Drive logins for the TV version of the app)

Does solid explorer works in fire stick

**Comment by Andrew-J-Larson** [comment]
> > Guys, stop bugging the dev. They already know the problem, and are working on a fix.
> > If you need a work around, I suggest using a file manager on your phone, such as [Solid Explorer](https://play.google.com/store/apps/details?id=pl.solidexplorer2), that has Google Drive connection features and video casting built-in. You can sign into your Google account in the app, and then cast the videos from there to your affected TV device, until this is sorted out.
> > (normally I would suggest using Solid Explorer directly on the TV, but the latest version of Google TV has broken Google Drive logins for the TV version of the app)
> 
> Does solid explorer works in fire stick

No clue, never used a Fire stick, ask a Fire stick related support forum elsewhere, like https://www.reddit.com/r/firetvstick/

**Comment by ldmania** [comment]
I guess that Google has planned no agreement to addons for streaming files from googledrive directly

**Comment by awsomjohn** [comment]
to prevent the proverbial spam that could come about me asking repeatedly if it's been fixed can somebody tag me or something when it doesso I don't have to spend another two days trying to figure out what I was doing wrong please and thank you

**Comment by Bhejan** [comment]
Anybuddy can solve kodi Google drive login issue please give me solution

**Comment by Bhejan** [comment]
Nplayer Google drive login issue  Couldn’t sign you in
This browser or app may not be secure. Learn more

Try using a different browser. If you’re already using a supported browser, you can try again to sign in.
Please solve this issue step by step 

!
![IMG_20240507_194143](https://github.com/cguZZman/plugin.googledrive/assets/168219217/88987300-5033-4b30-ac10-8e7cc83288f9)

**Comment by Liva1525** [comment]
> Nplayer Google drive login issue  Couldn’t sign you in
> This browser or app may not be secure. Learn more
> 
> Try using a different browser. If you’re already using a supported browser, you can try again to sign in.
> Please solve this issue step by step 
> 
> !
> ![IMG_20240507_194143](https://github.com/cguZZman/plugin.googledrive/assets/168219217/88987300-5033-4b30-ac10-8e7cc83288f9)
> 

Premium nplayer can be login easily maybe you are using moded nplayer

**Comment by Varun23102002** [comment]
Try VPN or uninstall update of Google app and Chrome Browser

**Comment by Varun23102002** [comment]
@Liva1525 
![IMG_20240508_074706_684](https://github.com/cguZZman/plugin.googledrive/assets/99336510/92fdc044-5dee-43ab-a10f-20fbb1794c2e)
![IMG_20240508_074711_127](https://github.com/cguZZman/plugin.googledrive/assets/99336510/94bd2e72-85a5-43d7-9b35-ced041f529cf)
![IMG_20240508_074504_042](https://github.com/cguZZman/plugin.googledrive/assets/99336510/7b4e3d67-eda8-41af-aea0-81d711c5d30b)
![IMG_20240508_074507_302](https://github.com/cguZZman/plugin.googledrive/assets/99336510/342b7195-4521-4c32-9f2d-7a1b40907953)

**Comment by Anshbhai27** [comment]
What about drivestream addon ?

**Comment by Bhejan** [comment]
> @Liva1525 ![IMG_20240508_074706_684](https://private-user-images.githubusercontent.com/99336510/328731747-92fdc044-5dee-43ab-a10f-20fbb1794c2e.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NDctOTJmZGMwNDQtNWRlZS00M2FiLWExMGYtMjBmYmIxNzk0YzJlLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlNWFhMzM1ODI1Nzg0ZTVlODdiMTA5ZWZjYjIyMGQ4MmFiNjFjM2I0ZjJlMTk4ODYxMjY2ZGM5YzIwOGNmNzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JLGd-8tlDUTZvx7ipkDKKlnEaTyL2PjAhtLbJf2P0Ac) ![IMG_20240508_074711_127](https://private-user-images.githubusercontent.com/99336510/328731755-94bd2e72-85a5-43d7-9b35-ced041f529cf.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NTUtOTRiZDJlNzItODVhNS00M2Q3LTliMzUtY2VkMDQxZjUyOWNmLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE0MzA2ZjRhYjkyMzA4ZWQxMWI4MzZjOTVmMmI3OThkZDc3ODZkNjA3N2Y4YTEzMzM3OWE1MmQxMWVkYmJhNDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2JpmNVfdbAxd72QD1EZceeSVEjP7MsPeUnDRyBYBwR8) ![IMG_20240508_074504_042](https://private-user-images.githubusercontent.com/99336510/328731761-7b4e3d67-eda8-41af-aea0-81d711c5d30b.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NjEtN2I0ZTNkNjctZWRhOC00MWFmLWFlYTAtODFkNzExYzVkMzBiLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiYmY5MzRiOTc0ODllYWNlZWY5MzdkYzRlOTZhOGMxNTdiNWZhM2YyMjMxNTNkM2U0OTFiMmY4ZTI4YWM3YTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GZSUqXsEdHy4J0_4Lm4pcYXTbm7MGkEqE64mHS0YQqA) ![IMG_20240508_074507_302](https://private-user-images.githubusercontent.com/99336510/328731765-342b7195-4521-4c32-9f2d-7a1b40907953.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NjUtMzQyYjcxOTUtNDUyMS00YzMyLTlmMmQtN2ExYjQwOTA3OTUzLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjNDgyNzk0Zjg5Njk1YWRlMDI4ZmZmYjg4ZWQxOGMxMDkxYTc1MDg5NDNhMThmYTdhMmM4NjYxN2YwZWEyYzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KCBPFkTyxrVRd8OeHbNEIjaWa9RbiVxZBtWdTkLppDU)

Please download link nplayer support Google drive login version

**Comment by Liva1525** [comment]
> > @Liva1525 ![IMG_20240508_074706_684](https://private-user-images.githubusercontent.com/99336510/328731747-92fdc044-5dee-43ab-a10f-20fbb1794c2e.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NDctOTJmZGMwNDQtNWRlZS00M2FiLWExMGYtMjBmYmIxNzk0YzJlLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlNWFhMzM1ODI1Nzg0ZTVlODdiMTA5ZWZjYjIyMGQ4MmFiNjFjM2I0ZjJlMTk4ODYxMjY2ZGM5YzIwOGNmNzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JLGd-8tlDUTZvx7ipkDKKlnEaTyL2PjAhtLbJf2P0Ac) ![IMG_20240508_074711_127](https://private-user-images.githubusercontent.com/99336510/328731755-94bd2e72-85a5-43d7-9b35-ced041f529cf.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NTUtOTRiZDJlNzItODVhNS00M2Q3LTliMzUtY2VkMDQxZjUyOWNmLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE0MzA2ZjRhYjkyMzA4ZWQxMWI4MzZjOTVmMmI3OThkZDc3ODZkNjA3N2Y4YTEzMzM3OWE1MmQxMWVkYmJhNDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2JpmNVfdbAxd72QD1EZceeSVEjP7MsPeUnDRyBYBwR8) ![IMG_20240508_074504_042](https://private-user-images.githubusercontent.com/99336510/328731761-7b4e3d67-eda8-41af-aea0-81d711c5d30b.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NjEtN2I0ZTNkNjctZWRhOC00MWFmLWFlYTAtODFkNzExYzVkMzBiLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiYmY5MzRiOTc0ODllYWNlZWY5MzdkYzRlOTZhOGMxNTdiNWZhM2YyMjMxNTNkM2U0OTFiMmY4ZTI4YWM3YTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GZSUqXsEdHy4J0_4Lm4pcYXTbm7MGkEqE64mHS0YQqA) ![IMG_20240508_074507_302](https://private-user-images.githubusercontent.com/99336510/328731765-342b7195-4521-4c32-9f2d-7a1b40907953.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NjUtMzQyYjcxOTUtNDUyMS00YzMyLTlmMmQtN2ExYjQwOTA3OTUzLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjNDgyNzk0Zjg5Njk1YWRlMDI4ZmZmYjg4ZWQxOGMxMDkxYTc1MDg5NDNhMThmYTdhMmM4NjYxN2YwZWEyYzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KCBPFkTyxrVRd8OeHbNEIjaWa9RbiVxZBtWdTkLppDU)
> 
> Please download link nplayer support Google drive login version 

I purchased from Play Store

**Comment by Liva1525** [comment]
> > > @Liva1525 ![IMG_20240508_074706_684](https://private-user-images.githubusercontent.com/99336510/328731747-92fdc044-5dee-43ab-a10f-20fbb1794c2e.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NDctOTJmZGMwNDQtNWRlZS00M2FiLWExMGYtMjBmYmIxNzk0YzJlLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlNWFhMzM1ODI1Nzg0ZTVlODdiMTA5ZWZjYjIyMGQ4MmFiNjFjM2I0ZjJlMTk4ODYxMjY2ZGM5YzIwOGNmNzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JLGd-8tlDUTZvx7ipkDKKlnEaTyL2PjAhtLbJf2P0Ac) ![IMG_20240508_074711_127](https://private-user-images.githubusercontent.com/99336510/328731755-94bd2e72-85a5-43d7-9b35-ced041f529cf.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NTUtOTRiZDJlNzItODVhNS00M2Q3LTliMzUtY2VkMDQxZjUyOWNmLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE0MzA2ZjRhYjkyMzA4ZWQxMWI4MzZjOTVmMmI3OThkZDc3ODZkNjA3N2Y4YTEzMzM3OWE1MmQxMWVkYmJhNDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2JpmNVfdbAxd72QD1EZceeSVEjP7MsPeUnDRyBYBwR8) ![IMG_20240508_074504_042](https://private-user-images.githubusercontent.com/99336510/328731761-7b4e3d67-eda8-41af-aea0-81d711c5d30b.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NjEtN2I0ZTNkNjctZWRhOC00MWFmLWFlYTAtODFkNzExYzVkMzBiLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiYmY5MzRiOTc0ODllYWNlZWY5MzdkYzRlOTZhOGMxNTdiNWZhM2YyMjMxNTNkM2U0OTFiMmY4ZTI4YWM3YTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GZSUqXsEdHy4J0_4Lm4pcYXTbm7MGkEqE64mHS0YQqA) ![IMG_20240508_074507_302](https://private-user-images.githubusercontent.com/99336510/328731765-342b7195-4521-4c32-9f2d-7a1b40907953.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNzI5NjcsIm5iZiI6MTcxNTE3MjY2NywicGF0aCI6Ii85OTMzNjUxMC8zMjg3MzE3NjUtMzQyYjcxOTUtNDUyMS00YzMyLTlmMmQtN2ExYjQwOTA3OTUzLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDEyNTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjNDgyNzk0Zjg5Njk1YWRlMDI4ZmZmYjg4ZWQxOGMxMDkxYTc1MDg5NDNhMThmYTdhMmM4NjYxN2YwZWEyYzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KCBPFkTyxrVRd8OeHbNEIjaWa9RbiVxZBtWdTkLppDU)
> > 
> > Please download link nplayer support Google drive login version 
> 
> I purchased from Play Store 

But not supported EAC3 codec audio track not need to purchased its an waste of money

**Comment by Bhejan** [comment]
Kodi Drive stream addon create how to step

**Comment by Bhejan** [comment]
Please solution Google drive movie watch on TV without casting

**Comment by Mfstick** [comment]
> What about drivestream addon ?

What addon is this? Can you share the addon link.

**Comment by satkader** [comment]
Hello, it happens to me too, is there an app for Google TV to see the drive content?

**Comment by Varun23102002** [comment]
> Hello, it happens to me too, is there an app for Google TV to see the drive content?

@satkader Use Mixplorer with Vimu media player Tv

**Comment by cguZZman** [comment]
Got a response from google yesterday (yes, it took this long... told ya) and the app is been reinstated and the verification is in progress. It can take long too, depending if they ask any additional stuff.

For the people asking about oauth: Yes, the addon uses OAuth only. Again, the addon uses **OAuth only** - no user, no passwords.

As I already commented before (many times) you can always setup your own [login page](https://drive-login.herokuapp.com/). The code is [published](https://github.com/cguZZman/drive-login) and you don't need mine. I made mine public to give back to Kodi, but this also means it need google's approval, for security.
If you don't know how to put your own server or you don't want to learn, then just wait. Or if you want, search for an alternative. 
You know how to set it up? Share it! Both things take time. I don't have it right now. Probably later.

**Comment by spacex97** [comment]
@cguZZman I am able to compile jar file from published drive-login code but can we have some readme as I think it need config file which is not automatically created and needs to be created manually and  I am little puzzled about it

**Comment by hrshlmhjn** [comment]
@cguZZman  I have non coding background little hard to understand. It will be good if anyone guides through it.

**Comment by aldacco** [comment]
I've replicated the login functionality and updated the authentication process. You can now use your own credentials for authentication. To enable this, you'll need to modify the source URL of the Google Drive add-on in Kodi to point to your app (https://kodi-drive-login.vercel.app). For detailed configuration instructions, please refer to the [repository](https://github.com/aldacco/drive-login#readme).

**Comment by Nickytun** [comment]
> I've replicated the login functionality and updated the authentication process. You can now use your own credentials for authentication. To enable this, you'll need to modify the source URL of the Google Drive add-on in Kodi to point to your app (https://kodi-drive-login.vercel.app). For detailed configuration instructions, please refer to the [repository](https://github.com/aldacco/drive-login#readme).

that's great. It's working. Thanks for your work

**Comment by Varun23102002** [comment]
@aldacco 
![IMG_20240520_123344](https://github.com/cguZZman/plugin.googledrive/assets/99336510/2e4620dc-db18-4c00-9ba9-30bc9ed8f432)
Thanks it's working

**Comment by Nickytun** [comment]
> I've replicated the login functionality and updated the authentication process. You can now use your own credentials for authentication. To enable this, you'll need to modify the source URL of the Google Drive add-on in Kodi to point to your app (https://kodi-drive-login.vercel.app). For detailed configuration instructions, please refer to the [repository](https://github.com/aldacco/drive-login#readme).

I try it and it's log in ok. But if I stop for a while. It's can't access to google drive. and I have to log in again. Pls check it

**Comment by Mfstick** [comment]
@aldacco Session is not maintaining. Its getting expired and showing error. Need to remove account and login to access gdrive

**Comment by cguZZman** [comment]
@aldacco thanks for sharing this.

Did a very quick review. First comments from a security point of view:
- Consider generate the pin and password using secure cryptographically methods. My implementation uses Java's SecureRandom, complaying with [RFC 1750](http://www.ietf.org/rfc/rfc1750.txt). I believe NodeJS's equivalent is https://nodejs.org/api/crypto.html#crypto_crypto_randombytes_size_callback. You can then call the `.toString('hex');` to mimic the same encoding.
- Consider removing owner's value defaulting to localhost ip. It's not secure since anybody could have access to the generated pin if they default to the same localhost ip.

**Comment by aldacco** [comment]
@cguZZman Yes, I will keep this in mind, as I have developed this solution as quickly as possible to solve this problem.

**Comment by aldacco** [comment]
> @aldacco Session is not maintaining. Its getting expired and showing error. Need to remove account and login to access gdrive

By the way I have already solved.


> I've replicated the login functionality and updated the authentication process. You can now use your own credentials for authentication. To enable this, you'll need to modify the source URL of the Google Drive add-on in Kodi to point to your app (https://kodi-drive-login.vercel.app). For detailed configuration instructions, please refer to the [repository](https://github.com/aldacco/drive-login#readme).

**Comment by hrshlmhjn** [comment]
@aldacco Thanks for the solution.

**Comment by m4cungg** [comment]
please help..!
![Screenshot_25](https://github.com/cguZZman/plugin.googledrive/assets/170401155/a21f2c7e-eaba-4f1e-8f41-192a3a09a8b2)

**Comment by Varun23102002** [comment]
@aldacco Showing server is temporary unavailable 
![IMG_20240521_123045244_HDR](https://github.com/cguZZman/plugin.googledrive/assets/99336510/89cb3f90-fd30-4193-8a96-af760ca7f46f)

**Comment by aldacco** [comment]
@m4cungg Delete all your accounts and log in again, a clarification I have removed the advanced options as it only works if you have your own instance. 
Since the issue is closed if you have problems you can make queries in the [repository.](https://github.com/aldacco/drive-login)

**Comment by m4cungg** [comment]
> @m4cunggHapus semua akun Anda dan masuk lagi, klarifikasi Saya telah menghapus opsi lanjutan karena ini hanya berfungsi jika Anda memiliki instance sendiri. Karena masalah telah ditutup, jika Anda mengalami masalah, Anda dapat mengajukan pertanyaan di repositori [.](https://github.com/aldacco/drive-login)

![Screenshot_27](https://github.com/cguZZman/plugin.googledrive/assets/170401155/39b1ae4c-401c-4356-a467-238da56a7dca)
can't login,stuck https://kodi-drive-login.vercel.app/callback

**Comment by javier191185** [comment]
> Got a response from google yesterday (yes, it took this long... told ya) and the app is been reinstated and the verification is in progress. It can take long too, depending if they ask any additional stuff.
> 
> For the people asking about oauth: Yes, the addon uses OAuth only. Again, the addon uses **OAuth only** - no user, no passwords.
> 
> As I already commented before (many times) you can always setup your own [login page](https://drive-login.herokuapp.com/). The code is [published](https://github.com/cguZZman/drive-login) and you don't need mine. I made mine public to give back to Kodi, but this also means it need google's approval, for security. If you don't know how to put your own server or you don't want to learn, then just wait. Or if you want, search for an alternative. You know how to set it up? Share it! Both things take time. I don't have it right now. Probably later.

First of all, thank you for your work. It was very useful this time.
Do you have some good news to us? Thank you again @cguZZman

**Comment by awsomjohn** [comment]
I guess I'm complete idiot then cuz I don't know what OAuth is is an app or
what?

**Comment by Hrk5259** [comment]
I got the solution..

Just download file explorer named X-plore. And u can simply login your gdrive and enjoy all content on your TV.

Hope it's working for all

**Comment by vishnuagbly** [comment]
I am still facing the same issue. Has anyone found any fix?

**Comment by rsxxxxx** [comment]
I signed up so I could tell you all how to fix this. I assume you're all using some variant of kodi/google tv/fire stick. For some reason, google drive isn't officially installable on a roku/google tv/onn box through the play store but go to apkmirror and grab the latest, sideload it and problem solved, full screen video works great.

All the apps and methods I tried that were mentioned did not work. I even tried a few other things but without a google workspace account (which isn't free) no third-party apps will work anymore. I don't know why google is doing this because it wasn't an issue in the past, also a few months ago they also started locking their google tv's (it's old news on xda that their pixel phones have never been locked). Good luck with your progress, @cguZZman I hope they approve it.

**Comment by TheTechRun** [comment]
> I signed up so I could tell you all how to fix this. I assume you're all using some variant of kodi/google tv/fire stick. For some reason, google drive isn't officially installable on a roku/google tv/onn box through the play store but go to apkmirror and grab the latest, sideload it and problem solved, full screen video works great.
> 
> All the apps and methods I tried that were mentioned did not work. I even tried a few other things but without a google workspace account (which isn't free) no third-party apps will work anymore. I don't know why google is doing this because it wasn't an issue in the past, also a few months ago they also started locking their google tv's (it's old news on xda that their pixel phones have never been locked). Good luck with your progress, @cguZZman I hope they approve it.

The method you provided does not work. "Sorry this app is not supported" on Android TV Boxes.

**Comment by rsxxxxx** [comment]
> > I signed up so I could tell you all how to fix this. I assume you're all using some variant of kodi/google tv/fire stick. For some reason, google drive isn't officially installable on a roku/google tv/onn box through the play store but go to apkmirror and grab the latest, sideload it and problem solved, full screen video works great.
> > All the apps and methods I tried that were mentioned did not work. I even tried a few other things but without a google workspace account (which isn't free) no third-party apps will work anymore. I don't know why google is doing this because it wasn't an issue in the past, also a few months ago they also started locking their google tv's (it's old news on xda that their pixel phones have never been locked). Good luck with your progress, @cguZZman I hope they approve it.
> 
> The method you provided does not work. "Sorry this app is not supported" on Android TV Boxes.

It was working, up until a few days ago. It installs but after opening, some prompts are cycled through the screen and then the error message displays.

**Comment by ademxorj** [comment]
Hello, is there any solution? If so, could you please guide me, thank you.

**Comment by Hrk5259** [comment]
Hi

You can install x-plore file manager to access Google drive. Kindly confirm
back if working..

**Comment by rsxxxxx** [comment]
> Hi You can install x-plore file manager to access Google drive. Kindly confirm back if working..

Doesn't work.

**Comment by Hrk5259** [comment]
But on my android tv .. it's working absolutely fine

**Comment by rsxxxxx** [comment]
I installed x-plore on my (unrooted) ONN 4k android box and google drive does not prompt. To make sure it wasn't me, I loaded several other included plugins and they came up. Except for one that says it's unauthorized now.  There are numerous complaints on r/androidtv about this app. To make matter worse: When I closed the app, it would not reopen again.

**Comment by MinhAmazing** [comment]
follow this guide and you can use google drive login normally: https://youtu.be/1piIB5DfuQo?si=tm8hT2okxm6cUDcX

**Comment by alarno** [comment]
> follow this guide and you can use google drive login normally: https://youtu.be/1piIB5DfuQo?si=tm8hT2okxm6cUDcX

Hi @MinhAmazing ,

I just wanted to take a moment to express my deepest gratitude for the incredible work you’ve done in fixing the Google login validation issue. Your solution not only resolved the problem perfectly but also saved me a tremendous amount of time and effort.

You’re truly a genius! Your expertise and attention to detail have made a world of difference, and I can’t thank you enough for your help. It’s clear that your contributions are invaluable, and I’m so appreciative of your support.

Thank you once again for your amazing work!

**Comment by greatestradioshow** [comment]
> follow this guide and you can use google drive login normally: https://youtu.be/1piIB5DfuQo?si=tm8hT2okxm6cUDcX

Is it possible that your video is no longer available/private?

**Comment by TheTechRun** [comment]
> > > I signed up so I could tell you all how to fix this. I assume you're all using some variant of kodi/google tv/fire stick. For some reason, google drive isn't officially installable on a roku/google tv/onn box through the play store but go to apkmirror and grab the latest, sideload it and problem solved, full screen video works great.
> > > All the apps and methods I tried that were mentioned did not work. I even tried a few other things but without a google workspace account (which isn't free) no third-party apps will work anymore. I don't know why google is doing this because it wasn't an issue in the past, also a few months ago they also started locking their google tv's (it's old news on xda that their pixel phones have never been locked). Good luck with your progress, @cguZZman I hope they approve it.
> > 
> > 
> > The method you provided does not work. "Sorry this app is not supported" on Android TV Boxes.
> 
> It was working, up until a few days ago. It installs but after opening, some prompts are cycled through the screen and then the error message displays.

It's fine. I came up with my own solution. Thanks

**Comment by alarno** [comment]
> > follow this guide and you can use google drive login normally: https://youtu.be/1piIB5DfuQo?si=tm8hT2okxm6cUDcX
> 
> 
> 
> Is it possible that your video is no longer available/private?

In fact youtube video told to configure google drive add on login to https://kodi.imminh.com/, but it looks it is down now...

**Comment by alarno** [comment]
> > follow this guide and you can use google drive login normally: https://youtu.be/1piIB5DfuQo?si=tm8hT2okxm6cUDcX
> 
> 
> 
> Hi @MinhAmazing ,
> 
> 
> 
> I just wanted to take a moment to express my deepest gratitude for the incredible work you’ve done in fixing the Google login validation issue. Your solution not only resolved the problem perfectly but also saved me a tremendous amount of time and effort.
> 
> 
> 
> You’re truly a genius! Your expertise and attention to detail have made a world of difference, and I can’t thank you enough for your help. It’s clear that your contributions are invaluable, and I’m so appreciative of your support.
> 
> 
> 
> Thank you once again for your amazing work!

Hi again, it looks https://kodi.imminh.com/ is down, any problem? Is there anything we can do?

**Comment by greatestradioshow** [comment]
> Hi again, it looks https://kodi.imminh.com/ is down, any problem? Is there anything we can do?

Is it not that the limit for the Google project is reached and therefore no new logins are possible?
I have to say the easiest way is to set up you own instance: https://github.com/aldacco/drive-login

**Comment by sureshsingh88** [comment]
> > Hi again, it looks https://kodi.imminh.com/ is down, any problem? Is there anything we can do?
> 
> Is it not that the limit for the Google project is reached and therefore no new logins are possible? I have to say the easiest way is to set up you own instance: https://github.com/aldacco/drive-login

Please create a video & share with community

**Comment by greatestradioshow** [comment]
@sureshsingh88 I'm on vacation right now, but if in 2 weeks this is stilled needed, I record a video tutorial.

**Comment by sureshsingh88** [comment]
Whole team is awaited your video

**Comment by greatestradioshow** [comment]
@sureshsingh88 

Modify the Drive add-on settings and try my app:

1 - Open Kodi and go to Add-ons > My add-ons > Video add-ons > Google Drive.
2 - Click on Configure.
3 - In the Advanced tab, find the Source property.
4 - Change the Source URL to: https://kodi-drive-login-psi.vercel.app/
5 - Save the changes.

Let me know if it works.

**Comment by awsomjohn** [comment]
This is what happens to me

**Comment by greatestradioshow** [comment]
Yes, I guess there is an issue with the OAuth which needs to be tackled first.
https://github.com/cguZZman/plugin.googledrive/issues/345#issuecomment-2391058188

**Comment by sureshsingh88** [comment]
> @sureshsingh88
> 
> Modify the Drive add-on settings and try my app:
> 
> 1 - Open Kodi and go to Add-ons > My add-ons > Video add-ons > Google Drive. 2 - Click on Configure. 3 - In the Advanced tab, find the Source property. 4 - Change the Source URL to: https://kodi-drive-login-psi.vercel.app/ 5 - Save the changes.
> 
> Let me know if it works.

Not working for me

**Comment by awsomjohn** [comment]
As I am not a programmer you're getting just a little far out of my league
best I can do is follow detailed instructions I took a shot because I could
follow that directions before but I don't know what you're talking about now

**Comment by tsdq512** [comment]
> Please guys comment the alternative way to play Google drive video files in phone and Tv

Air Explorer, RaiDrive

**Comment by preeti150490** [comment]
> [@aldacco](https://github.com/aldacco) Session is not maintaining. Its getting expired and showing error. Need to remove account and login to access gdrive

Thankyou it worked for me

**Comment by Axanderism** [comment]
> > [@aldacco](https://github.com/aldacco) Session is not maintaining. Its getting expired and showing error. Need to remove account and login to access gdrive
> 
> Thankyou it worked for me

What worked for you, please?

**Comment by Axanderism** [comment]
> I've replicated the login functionality and updated the authentication process. You can now use your own credentials for authentication. To enable this, you'll need to modify the source URL of the Google Drive add-on in Kodi to point to your app (https://kodi-drive-login.vercel.app). For detailed configuration instructions, please refer to the [repository](https://github.com/aldacco/drive-login#readme).

Looking into this, but is there an even more basic step-by-step guide anywhere? The first step asks me to do with something with next.js but I have no idea what next.js is.

**Comment by preeti150490** [comment]
> > > [@aldacco](https://github.com/aldacco) Session is not maintaining. Its getting expired and showing error. Need to remove account and login to access gdrive
> > 
> > 
> > Thankyou it worked for me
> 
> What worked for you, please?

Already replied to ur query hope it helps n u too r able to view ur files on tv all the best

**Comment by Axanderism** [comment]
> [@Liva1525](https://github.com/Liva1525) why are you spamming?
> 
> What should I copy to a fresh Kodi installation from my Kodi with authenticated and working Google Drive addon to make it work there? It is possible or will it make some conflicts when using addon on 2 devices at the same time?
> 
> UPDATE:
> 
> I copied a _accounts.db_ file from _Kodi/userdata/addon_data/plugin.googledrive_ and it works. So if you have verified account somewhere, e.g. on your PC, and want to use this account on another device, then you can put there this file. I haven't checked what will happend if I use that account on 2 devices at the same time. If this account will be verified separately it propably would work, but when I have copies, it may cause problems.

Just tried copying the accounts file from an old Kodi on an old Windows tablet and pasting it into Android. It reinstates my account as logged in, but any attempt to access files or folders returns an error. 

Thought it sounded too good to be true!

**Comment by kamillopl** [comment]
> Just tried copying the accounts file from an old Kodi on an old Windows tablet and pasting it into Android. It reinstates my account as logged in, but any attempt to access files or folders returns an error.
> 
> Thought it sounded too good to be true!

I'm not lying.
I've copied this file to Xbox and Google TV Kodi instances and it just works fine. Are this accounts working on original Kodi instalation?

There is open issue with error message when accessing files, I've the same situation, but on Fire Stick with same account but different login session. Maybe there is another issue related with something else and you got that. In my case Fire Stick wasn't used for few months so account verification could flew away, but I remember that in case like this was different error message, something with lost session.

Anyway I'll try use accounts.db file from Fire Stick somewhere else to check if the error will disappear or something is wrong with verification of this account.

For you I reccomend to reinstall Kodi, install Google Drive addon, paste advancedsettings.xml and accounts.db file in proper destinations. If it works on original Kodi installation it should work on different. If not, check the versions of addon and get the same or copy whole addon.

**Comment by Axanderism** [comment]
> > Just tried copying the accounts file from an old Kodi on an old Windows tablet and pasting it into Android. It reinstates my account as logged in, but any attempt to access files or folders returns an error.
> > Thought it sounded too good to be true!
> 
> I'm not lying. I've copied this file to Xbox and Google TV Kodi instances and it just works fine. Are this accounts working on original Kodi instalation?
> 
> There is open issue with error message when accessing files, I've the same situation, but on Fire Stick with same account but different login session. Maybe there is another issue related with something else and you got that. In my case Fire Stick wasn't used for few months so account verification could flew away, but I remember that in case like this was different error message, something with lost session.
> 
> Anyway I'll try use accounts.db file from Fire Stick somewhere else to check if the error will disappear or something is wrong with verification of this account.
> 
> For you I reccomend to reinstall Kodi, install Google Drive addon, paste advancedsettings.xml and accounts.db file in proper destinations. If it works on original Kodi installation it should work on different. If not, check the versions of addon and get the same or copy whole addon.

Not suggesting you're lying, just that there are other factors at play that make this method more complex than just copying a single file from an old installation.

**Comment by Hrk5259** [comment]
There is no way to login now.. i am also still waiting for solution
